packages/jaeger-ui/src/components/DependencyGraph/JaegerForceGraphArrowLink.js

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
A couple of requests for this file.

Can you update the 2017 to 2019 in the copyright?

We're aiming to use TypeScript for all new files. Would you mind converting to *.tsx?

Also, I am thinking the `Jaeger` scoping of `JaegerForceGraphArrowLink` is kind of implied; what do you think of renaming to just `ForceGraphArrowLink`?